### PR TITLE
Update README to fix a few issues

### PR DESCRIPTION
**Pull request by eweiter**
There were a few issues in the README that needed to be fixed.

### CODE REVIEW

**Comment by eweiter** [comment]
@beemarie I copied and pasted that from the IBM Cloud docs and forgot to change it the first time.  It shouldn't be there as it won't work on Github anyways.

File: README.md [hunk 1]
```diff
@@ -32,27 +32,24 @@ The web form will then retrieve back the image using the *setCurrentProfileImage
...
+   
+   * **Create an new instance**: Selecting this option will take you to the IBM Cloud page for creating Cloud Object Storage instances. It is important that after you create your instance you create a set of Service Credentials that contain the needed HMAC keys (See below)
+   * **Input your own credentials**: Selecting this will prompt you to manually enter your own credentials for a COS instance
+   * **Existing Instances**: If you already have any COS instances created they should be automatically populated in the dropdown. Clicking an existing instance will attempt to fetch the credentials as well as any Buckets existing on that instance
+   ****Important Note****
+   In order for the COS Template to be deployed properly your COS instance should have HMAC keys present as well as an already existing bucket.  For information on creating HMAC keys refer to this documentation: [Create COS Service Credentials](https://console.bluemix.net/docs/services/cloud-object-storage/iam/service-credentials.html#service-credentials)
```
Comment (beemarie) [inline README.md]: "For the COS Template to be deployed properly, your COS instance should have..."  For vs in order for & a comma after properly.

File: README.md [hunk 2]
```diff
@@ -32,27 +32,24 @@ The web form will then retrieve back the image using the *setCurrentProfileImage
...
 
-  5. Once you input the information for your COS instance the **Deploy** button should be enabled and you can deploy the template. 
+  3. Once you input the information for your COS instance the **Deploy** button should be enabled and you can deploy the template.
```
Comment (beemarie) [inline README.md]: "enabled, and you can" . comma after enabled.

File: README.md [hunk 3]
```diff
@@ -32,27 +32,24 @@ The web form will then retrieve back the image using the *setCurrentProfileImage
...
 
-  5. Once you input the information for your COS instance the **Deploy** button should be enabled and you can deploy the template. 
+  3. Once you input the information for your COS instance the **Deploy** button should be enabled and you can deploy the template. 
 
-  6. After the template deploys you should be on the **Code** page for the *app* Action. You can then click on the **Endpoints** tab in the lefthand navigation. Under the **Endpoints** tab you will find a web action section with a link. Copy this link without the .json suffix, and paste it into your browser's address bar. This will display the template's web form for you to interact with. 
+  4. After the template deploys you should be on the **Code** page for the *app* Action. You can then click on the **Endpoints** tab in the lefthand navigation. Under the **Endpoints** tab you will find a web action section with a link. Copy this link without the .json suffix, and paste it into your browser's address bar. This will display the template's web form for you to interact with.
```
Comment (beemarie) [inline README.md]: "template deploys, you should be on the"  comma after deploys?

**Pull request merged**